cluster/gce/gci/configure-helper.sh

```diff
@@ -554,7 +554,7 @@ function start-kubelet {
   flags+=" --cluster-dns=${DNS_SERVER_IP}"
   flags+=" --cluster-domain=${DNS_DOMAIN}"
   flags+=" --config=/etc/kubernetes/manifests"
-  flags+=" --experimental-mounter-path=${KUBE_HOME}/bin/mounter"
+  flags+=" --experimental-mounter-path=${KUBE_HOME}/bin/newmounter/mounter"
```

Comment:
I am thinking normally .../bin/ should only container executable files. 
Maybe put all the docker image files to /home/kubernetes/mounter/rootfs to make it more clear?